pjlib/src/pj/ssl_sock_ossl.c

```diff
@@ -1695,7 +1695,12 @@ static void ssl_destroy(pj_ssl_sock_t *ssock)
 /* Reset SSL socket state */
 static void ssl_reset_sock_state(pj_ssl_sock_t *ssock)
 {
-    ossl_sock_t *ossock = (ossl_sock_t *)ssock;
+    ssl_reset_sock_state_with_error(ssock, PJ_TRUE);
+}
+
+static void ssl_reset_sock_state_with_error(pj_ssl_sock_t* ssock, pj_bool_t check_error)
+{
+    ossl_sock_t* ossock = (ossl_sock_t*)ssock;
```

Comment:
Minor: our PointerAlignment is actually `Right`, i.e. see the initial code:
`ossl_sock_t *ossock = (ossl_sock_t *)ssock`